ansible/vars/topo_t0-isolated-d128u128s2.yml

```diff
@@ -829,7 +829,7 @@ configuration:
       Ethernet1:
         ipv4: 10.0.0.65/31
         ipv6: fc00::82/126
-    bp_interfaces:
+    bp_interface:
```

Comment:
do you mind to also get d128u128s1 updated?

Reply:
done